.github/workflows/kind.yml

```diff
@@ -436,7 +436,7 @@ jobs:
         run: |
           mkdir log
           mkdir test-e2e-fa-coverage
-          ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-fa-coverage ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage --flow-visibility --ip-family dual
+          ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-fa-coverage ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage --flow-visibility --ip-family ds
```

Comment:
why is it changed from the correct value to a wrong one? I believe `ds` can't get a dual stack cluster: https://kind.sigs.k8s.io/docs/user/configuration/#networking

Reply:
Hi @tnqn , my bad I didn't notice the kind configuration file I will correct it. actually in the kind builder I am passing IP_MODE to `test.sh `script and in this script there is a condition 

`--I-mode must be ds` and other scripts have `dual` value. I thought it should be either `ds` or `dual` but the correct value is `dual` so it should be `dual` everywhere. could you please correct me?

`if [[ "${IP_MODE}" != "${DEFAULT_IP_MODE}" && "${IP_MODE}" != "ipv6" && "${IP_MODE}" != "ds" ]]; then
    echoerr "--ip-mode must be ipv4, ipv6 or ds"
    exit 1
fi`

I didn't change `test.sh` because this script used by many jobs in jenkins.